wgsl/index.bs

```diff
@@ -395,21 +411,103 @@ The following types are <dfn dfn noexport>host-shareable</dfn>:
 * [[#array-types]] if the array type has a stride attribute and its element type is host-shareable
 * [[#struct-types]] if all its members are host-shareable
```

Comment:
This list is essentially the same  as IO-shareable, but with the addition of runtime-sized arrays?
Perhaps, it's more clear to specify the difference directly?

Reply:
I'll add a note highlighting the difference.